diff --git a/node_modules/react-native-safe-area-context/lib/module/SafeAreaContext.js b/node_modules/react-native-safe-area-context/lib/module/SafeAreaContext.js
index 0a8e5d9..b6b4d9a 100644
--- a/node_modules/react-native-safe-area-context/lib/module/SafeAreaContext.js
+++ b/node_modules/react-native-safe-area-context/lib/module/SafeAreaContext.js
@@ -1,6 +1,6 @@
 import React, { createContext, useContext, useEffect, useRef, useState } from 'react';
 import { Dimensions, StyleSheet, View } from 'react-native';
-import NativeSafeAreaProvider from './NativeSafeAreaProvider';
+import NativeSafeAreaProvider from './specs/NativeSafeAreaProvider';
 import { initialWindowMetrics } from './InitialWindowMetrics';
 
 const isDev = process.env.NODE_ENV !== 'production';
